app/src/main/res/layout/checkbox_left_sub_preference.xml

```diff
@@ -12,7 +12,7 @@
     android:clickable="true"
     android:focusable="true"
     android:gravity="center_vertical"
-    android:layout_marginStart="@dimen/custom_checkbox_alignment_margin">
+    android:paddingStart="@dimen/custom_checkbox_alignment_margin">
```

Comment:
I think this makes sense to me, but wondering why did we decide to make this change?

Reply:
For the clickable area; with margin, this setting and Search -> Show in private sessions were missing considerable tappable area on the left.

Reply:
Normally, we should probably make this change in its own issue/PR.